app/storage.js

```diff
@@ -1,4 +1,8 @@
 function isUserLoggedIn() {
-    const user = localStorage.getItem('user');
-    return user !== null ? user : false;
-}
\ No newline at end of file
+    const user = localStorage.getItem('username');
+    return true; //user !== null;
```

Comment:
Esto regresalo como estaba.
```suggestion
    return user !== null;
```